test/chunkserver/mock_copyset_node.h

```diff
@@ -43,6 +43,7 @@ class MockCopysetNode : public CopysetNode {
     MOCK_METHOD0(Run, int());
     MOCK_METHOD0(Fini, void());
     MOCK_CONST_METHOD0(IsLeaderTerm, bool());
+    MOCK_CONST_METHOD1(IsLeaseLeader, bool(const braft::LeaderLeaseStatus&));
```

Comment:
since we have upgraded gtest to v1.12, we can use new `MOCK_METHOD`


```suggestion
    MOCK_METHOD(bool, IsLeaseLeader, (const braft::LeaderLeaseStatus&), (const, override));
```